### ISSUE TITLE
Update data for Edge 13 for new criteria

### ISSUE THREAD

**Issue by dstorey** [stevefaulkner/HTML5accessibility#45]
I have a mostly complete test run for this. Just double checking some tests.Will make a PR proposal later this week with that data.

Note this isn't Edge 14 (currently in development), which includes the new accessibility work. The test only includes released browsers.

**Comment by dstorey** [comment]
Noting issues in Edge 13 here, so can file public bugs later. Most already have internal bugs, but easier to create new ones than try to make the internal ones public.

Name/Description:

* aria-label doesn't work as expected. Puts element in A11y tree, but uses text contents of element rather than the element that the ID points to.
* aria-describedby and title don't work for AccDesc as there is nothing in UIA for accessible description prior to Win 10 Anniversary updated.

Figure/Figcaption

* figcaption not used as accessible name of figure
* figcaption not mapped (not sure if this is covered in general landmarks bug I filed already

Text level

* mark doesn't have correct name/desc. Basically the same as the other elements so can probably use same bug. Title isn't used as accname in this case, but I think that is bug in spec it overrides text content (spec doesn't mention using text content in accName)

Canvas

* Canvas is no longer mapped in Edge. (works in Edge 14, so not sure what happened here)

Video

* Regression with CC sub menu and volume control not responding to keyboard once opened. Works in IE11 and Edge14, so again, not sure what is happening here

Controls:

* Error states not exposed (they do set invalidforform so could arguably be partial here). Fixed in Edge 14 too.

Date related

* some of the mapping doesn't look like it exists. only exposes the fields are regular edits, but the date UI is exposed

Datalist

* datalist mapped as list, but no controllerFor on parent input and doesn't map that input as combobox

Hidden

* Regression where hidden elements are in the UIA tree (they are not in IE11), but it does map to isOffscreen, but that is not the spec.

Placeholder

* There is no ariaPlaceholder in UIA in pre Windows 10 Anniversary, so doesn't map correctly even though placeholder is used correctly as accessible name.
* Placeholder not used as accDesc as no concept in pre Win10 Anniversary 
* placeholder doesn't beat title for accName
* Aria-placeholder doesn't exist so it doesn't win when empty placeholder

**Comment by stevefaulkner** [comment]
I beleive this has been done with https://github.com/stevefaulkner/HTML5accessibility/pull/50
